data/alpha/field/jobcenter-group.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: jobcentre-group
+phase: alpha
+register: 'jobcentre-group'
+text: 'A jobcentre office group'
```

Comment:
does not read right, maybe a grouping of jobcentre offices?

Reply:
ammended